src/codegen.cpp

```diff
@@ -4889,7 +4889,7 @@ extern "C" void jl_fptr_to_llvm(jl_fptr_t fptr, jl_lambda_info_t *lam, int specs
 #define V128_BUG
 #endif
 
-static void init_julia_llvm_env(Module *m)
+static void init_julia_llvm_meta()
```

Comment:
I think the c++compat warning is only for C code (and we have many `()` in C++ function declarations/definitions) but sure....